graspy/inference/latent_distribution_test.py

```diff
@@ -170,10 +236,46 @@ def _embed(self, A1, A2):
             X1_hat = np.concatenate(X1_hat, axis=-1)
             X2_hat = np.concatenate(X2_hat, axis=-1)
         elif isinstance(X1_hat, tuple) ^ isinstance(X2_hat, tuple):
-            raise ValueError("Input graphs do not have same directedness")
+            msg = (
+                "input graphs do not have same directedness. "
+                "consider symmetrizing the directed graph."
+            )
+            raise ValueError(msg)
 
         return X1_hat, X2_hat
 
+    def _sample_modified_ase(self, X, Y, pooled=False):
```

Comment:
pooled never gets used, then? since not a kwarg for the class? not sure whether to have this as a kwarg but if not, what is it doing?

Reply:
you are correct that it is impossible to set, unless one goes into source code. i had it there because i was trying both, but has no conclusive evidence that using a pooled one has any benefit.

i predominantly decided to leave it in the code so that it would be easier to whoever decides to extensively study the behavior when using and not using it, similarly to how we had code for directed that's never getting used.

i don't mind allowing to set it as a class kwarg. the primary reason why i didn't is because i wanted to avoid confusion for users. there is nothing in the literature on using this, other than a line in my manuscript that says that hypothetically one can do so.

i am happy with either of the following three options:
- leave it as is (or leave it in commented out. which, for practical reasons is equivalent)
- make it a class kwarg
- remove it completely

Reply:
my vote is make it a class kwarg, give an explanation in the docs, explain the default, and say that the not default way is untested/experimental

Reply:
sounds fair to me. doing dis now.

Reply:
fixed. feel free to change the wording.